Automattic/wp-calypso
# Revert "FSE: Add API endpoint to side-load images"

Reverts Automattic/wp-calypso#34823

#### Testing instructions

1. Try to edit the existing page or create a new one.
2. Verify that you are no longer seeing fatal errors.
